gen/dcos-config.yaml

```diff
@@ -526,6 +526,11 @@ package:
   - path: /etc_master/marathon
     content: |
       MARATHON_ZK=zk://zk-1.zk:2181,zk-2.zk:2181,zk-3.zk:2181,zk-4.zk:2181,zk-5.zk:2181/marathon
+{% switch dcos_overlay_enable %}
+{% case "true" %}
+      MARATHON_CMD_DEFAULT_NETWORK_NAME={{ dcos_overlay_network_default_name }}
```

Comment:
I posed a similar question. I didn't get answers from anyone. Is there someone on product @spahl that might have an opinion?